funnel/templates/js/update.js.jinja2

```diff
@@ -26,24 +26,24 @@
               <ul class="bullet-separated-list--flex mui--text-light mui--text-body2" v-if="update.number">
                 <li><a class="text-uppercase mui--text-light nounderline" :href="update.urls.view">{{ gettext('Update') }} #{{ update.number }}</a></li>
                 <li v-if="age"><a class="mui--text-light nounderline" :href="update.urls.view">{{ age }}</a></li>
-                <li v-if="update.visibility == 'participants'">
-                  <faicon :icon="'lock-alt'"></faicon>
-                </li>
-                <li v-if="update.visibility == 'members'">
-                  <faicon :icon="'lock-alt'"></faicon>
-                </li>
-                <li v-if="update.is_pinned">
-                  <a v-if="iseditor" href="#" @click="handlePinEvent($event, `pin-form-${update.uuid_b58}`, update.urls.edit);">
-                    <faicon :icon="'thumbtack-solid'" :baseline=false :css_class="'mui--align-middle'"></faicon>
-                  </a>
-                  <faicon v-else :icon="'thumbtack-solid'" :baseline=false :css_class="'mui--align-middle'"></faicon>
-                </li>
-                <li v-if="!update.is_pinned && iseditor">
-                  <a href="#" @click="handlePinEvent($event, `pin-form-${update.uuid_b58}`, update.urls.edit);" class="update__heading__unpin">
-                    <faicon :icon="'thumbtack'" :baseline=false :css_class="'mui--align-middle'"></faicon>
-                  </a>
-                </li>
               </ul>
+              <li v-if="update.visibility == 'participants'">
```

Comment:
`<li>` shouldn't be outside the `<ul>`. Instead, change the `<ul><li>` to have no bullet, then add a style for adding it back and apply that style to the second `<li>` (for age).